pertpy/preprocessing/_guide_rna.py

```diff
@@ -109,6 +112,93 @@ def assign_to_max_guide(
 
         return None
 
+    def assign_mixture_model(
+        self,
+        adata: AnnData,
+        model: str = "Poisson_Gauss_Mixture",
+        output_key: str = "assigned_guide",
+        no_grna_assigned_key: str = "Negative",
+        max_assignments_per_cell: int = 5,
+        multiple_grna_assigned_key: str = "Multiple",
+        multiple_grna_assignment_string: str = "+",
+        only_return_results: bool = False,
+        uns_key: str = "guide_assignment_params",
+        verbose: bool = False,
+        **kwargs,
+    ) -> np.ndarray | None:
+        """Assigns gRNAs to cells using a mixture model.
+
+        Args:
+            adata: Annotated data matrix containing gRNA values
+            model: The model to use for the mixture model. Currently only `Poisson_Gauss_Mixture` is supported.
+            output_key: Assigned guide will be saved on adata.obs[output_key]. default value is `assigned_guide`.
+            no_grna_assigned_key: The key to return if no gRNA is expressed enough.
+            max_assignments_per_cell: The maximum number of gRNAs that can be assigned to a cell.
+            multiple_grna_assigned_key: The key to return if multiple gRNAs are assigned to a cell.
+            multiple_grna_assignment_string: The string to use to join multiple gRNAs assigned to a cell.
+            only_return_results: If True, input AnnData is not modified and the result is returned as an np.ndarray.
+            verbose: If True, shows progress bar.
+            kwargs: Are passed to the mixture model.
```

Comment:
OK, then maybe 
```suggestion
            mixture_model_kwargs: Are passed to the mixture model.
```
Can we also specifiy that this is scikit-learn or so? It should be clear which implementation.